db/schema.rb

```diff
@@ -11,7 +11,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20181715075510) do
+ActiveRecord::Schema.define(version: 20182030152200) do
```

Comment:
we still need to rebase the [patch](https://github.com/kubic-project/velum/blob/master/packaging/suse/patches/0_set_default_salt_events_alter_time_column_value.rpm.patch) when changing the schema. example -> https://github.com/kubic-project/velum/pull/524/commits/4ec7dbab1666f77a1b236a65e1d2e83b5f80bcb3

Reply:
Have updated the patch.